.github/pull_request_template.md

```diff
@@ -1,14 +1,25 @@
-## Overview: ##
+## Overview
 
-## Related Jira tickets: ##
+
+
+## Related
 
 * [FP-123](https://jira.tacc.utexas.edu/browse/FP-123)
+<!--
+- requires|required by|mimics https://github.com/TACC/.../pull/...
+-->
```

Comment:
@wesleyboar Is this intentional?

Reply:
Yes. Just a suggestion only that can be dropped or improved.

I use it in my PRs to reference other PRs and explain relationship succinctly.

<details><summary>Examples</summary>

|  |  |
| - | - |
| ![Screen Shot 2022-06-13 at 11 43 34](https://user-images.githubusercontent.com/62723358/173403538-319e5ed2-2bee-46a8-8e3a-36b1e81578d8.png) | ![Screen Shot 2022-06-13 at 11 43 51](https://user-images.githubusercontent.com/62723358/173403541-5b6edfcb-072d-49bb-b0e5-ca4458404f1f.png) |
| ![Screen Shot 2022-06-13 at 11 44 06](https://user-images.githubusercontent.com/62723358/173403544-d17a33df-5a97-4865-a872-d48595fb54b0.png) | ![Screen Shot 2022-06-13 at 11 45 52](https://user-images.githubusercontent.com/62723358/173403805-af89a11c-6b09-4018-93f3-988dc0d37420.png) |

</details>

Reply:
I like those examples. For this template though, my feeling is simpler is better. Perhaps drop?

Reply:
Okay 👍 .